### ISSUE TITLE
Still need a relaunch for the Head model

### ISSUE THREAD

**Issue by Paroyer** [CWTesseract/LiveCubReload#1]
Work great when loading some model, but like when i copy over all the goblin file for the Feline custom race the head is always invisible till next relaunch

**Comment by Andoryuuta** [comment]
Thanks for the report!

Somebody else mentioned this and I have looked into it a little bit over the past couple of days, but I still haven't found the issue.

For some reason the game seems to be treating head models differently than the rest of the sprites. It also seems to be the playable race head models in specific, because you can use something like Geo's cheat table to change your current creature's head model to 500 (cow head), and update the `cow-head.cub` live just fine.

**Comment by Paroyer** [comment]
Yeah it work for npc wonder for pet tho imma test it, its weird cause except this everything work fine like switching boat glider and such live really just the head x)
